### PR TITLE
Update abstract-wc-data.php

### PR DESCRIPTION
**Pull request by jamsheerkrd**
https://github.com/woocommerce/woocommerce/issues/21640
https://github.com/woocommerce/woocommerce/issues/21664
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:
Fixed the issue as per the issues submited https://github.com/woocommerce/woocommerce/issues/21640 myself

Closes # .

### CODE REVIEW

File: includes/abstracts/abstract-wc-data.php [hunk 1]
```diff
@@ -445,7 +445,7 @@ public function update_meta_data( $key, $value, $meta_id = 0 ) {
 			}
 		}
 
-		if ( $array_key ) {
+		if ( $array_key !== false ) {
```
Comment (claudiulodro) [inline includes/abstracts/abstract-wc-data.php]: [This should use a yoda condition](https://travis-ci.org/woocommerce/woocommerce/jobs/445539921#L956) (`false !== $array_key`)

**Review by claudiulodro — CHANGES_REQUESTED** [review]
Hi. Good catch and thanks for the contribution! This fix looks good; just needs one small tweak to fix a code style issue.

File: includes/abstracts/abstract-wc-data.php (hunk 1 above)
Comment (jamsheerkrd) [reply includes/abstracts/abstract-wc-data.php]: I have updated it, Eagerly waiting to see the changes live as this bug affecting our woocommerce extension.

**Review by claudiulodro — APPROVED** [review]
Thanks for the quick turnaround on that change! This looks good to go.

**Pull request merged**